### DISCUSSION TITLE
Activity feed entries show 3 times

### DISCUSSION THREAD

**Discussion by codecat** [community/community#4078] [Other Feature Feedback, Questions, & Ideas]
I've been noticing this for a long time; I see triple entries in my activity feed, all of the same activity and the same user. It seems to happen frequently, but not always. It happens for all types, subscription starts & cancellations.

Hopefully this is the right place to report bugs.

![image](https://user-images.githubusercontent.com/136534/120495644-30ce6500-c3bd-11eb-984a-6832f0ee3234.png)

**Comment by bdresser** [comment]
Thanks @codecat! We just shipped a patch to fix this – let us know if you continue to experience this issue :)
